custom_components/tuya_local/devices/tion_breezer_bio_x.yaml

```diff
@@ -0,0 +1,237 @@
+name: Tion Breezer Bio X
+products:
+  - id: 9mqdhwklpvnnvb7t
+entities:
+  # Main entities
+  - entity: switch
+    name: "Power"
+    icon: "mdi:power"
+    dps:
+      - id: 101
+        name: switch
+        type: boolean
+  - entity: number
+    name: "Speed"
+    icon: "mdi:fan"
+    dps:
+      - id: 102
+        name: value
+        type: integer
+        range:
+          min: 1
+          max: 7
+  - entity: number
+    name: "Heater"
+    icon: "mdi:thermometer-plus"
+    dps:
+      - id: 114
+        name: value
+        type: integer
+        unit: C
+        range:
+          min: 0
+          max: 30
+  - entity: sensor
+    name: "Indoor Temperature"
+    class: temperature
+    dps:
+      - id: 108
+        name: sensor
+        type: integer
+        unit: "°C"
+  - entity: sensor
+    name: "Outdoor Temperature"
+    class: temperature
+    dps:
+      - id: 112
+        name: sensor
+        type: integer
+        unit: "°C"
+  - entity: sensor
+    name: "Filter Life"
+    icon: "mdi:air-filter"
+    dps:
+      - id: 115
+        name: sensor
+        type: integer
+        unit: "%"
+  - entity: sensor
+    name: "CO2"
+    class: carbon_dioxide
+    dps:
+      - id: 106
+        name: sensor
+        type: integer
+        unit: "ppm"
+  - entity: sensor
+    name: "PM2.5"
+    class: pm25
+    dps:
+      - id: 116
+        name: sensor
+        type: integer
+        unit: "µg/m³"
+  - entity: sensor
+    name: "Humidity"
+    class: "humidity"
+    dps:
+      - id: 119
+        name: sensor
+        type: integer
+        unit: "%"
+
+  # Configuration entities
+  - entity: switch
+    name: "Night Mode"
+    category: config
+    icon: "mdi:weather-night"
+    dps:
+      - id: 132
+        name: switch
+        type: boolean
+  - entity: switch
+    name: "Child Lock"
+    category: config
+    icon: "mdi:hand-back-right-off"
+    dps:
+      - id: 111
+        name: switch
+        type: boolean
+  - entity: select
```

Comment:
This should be a light entity, with dp name brightness, and values mapped to 0, 128, 255